src/styles/sidebar/components/tag-editor.scss

```diff
@@ -0,0 +1,45 @@
+@use "../../mixins/forms";
+@use "../../mixins/buttons";
+@use "../../variables" as var;
+
+.tag-editor {
+  &__input {
+    &::-webkit-calendar-picker-indicator {
+      // Turn off the default arrow icon
```

Comment:
Hmm ... do we really want to disable this affordance?

Reply:
We don't have control over this styling too much, it just shows on hover somewhat abruptly and and Lyza has some concerns about that. I'll remove it so we can see it again.